docker-compose.yml

```diff
@@ -1,7 +1,31 @@
-MongoDB:
-  image: sameersbn/mongodb:latest
-  ports:
-    - "27017:27017"
-  volumes:
-    - /srv/docker/mongodb:/var/lib/mongodb
-  restart: always
+version: '3'
+
+services:
+  adopteitor_api:
+    env_file:
+      - .env
+    container_name: adopteitor_api
```

Comment:
This will read the .env file and export the variables. Right now it's just `ENV`, and when it's empty it defaults to dev mode.